.github/workflows/image_metadata.yml

```diff
@@ -0,0 +1,68 @@
+name: Image metadata check CI
+
+on:
+  #push:
+  #  branches: [master]
+  pull_request:
+    paths:
+      - '**.webp'
+      - '**.png'
+      - '**.jpg'
+      - '**.jpeg'
+  
+jobs:
+  build:
+    name: Ubuntu - Image Metadata
+    runs-on: ubuntu-latest
+    
+    permissions:
+      contents: read
+      pull-requests: write
+
+    env:
+      SHA: ${{ github.event.pull_request.head.sha }}
+      BASE_SHA: ${{ github.event.pull_request.base.sha }}
+
+    steps:
+      - name: exiftool installation
+        run: |
+          sudo apt-get install --assume-yes exiftool
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: read/export image metadata
+        run: |
+          mapfile -t image_files < <(git diff --name-only ${BASE_SHA} ${SHA} | grep '\.webp$')
```

Comment:
Why is this just looking for webp files?

Reply:
That was a test that shouldn't have gotten in there...